samples/subsys/ipc/openamp/src/main.c

```diff
@@ -292,6 +300,14 @@ void main(void)
 	defined(CONFIG_SOC_V2M_MUSCA_B1)
 	wakeup_cpu1();
 	k_msleep(500);
+#elif defined(CONFIG_SOC_QEMU_CORTEX_A53)
+/* OPENAMP_CLIENT_START_ADDR is the ＿start symbole of remote */
+#define OPENAMP_CLIENT_START_ADDR 0x7a000ffcU
```

Comment:
How could you make sure the `_start` entry is always `0x7a000ffcU`? Is that better to get this from dts?

Reply:
I am figuring out not using magic number  but a symbol for the entry of client. The entry of client can be determined after the final linking of client. Then, the entry of client can be given to the host.

To make things simple,  I just use a predefined number.

Another approach is to use resource table,  but not covered in subsys/ipc/openamp example

Reply:
@povergoing already fixed, use the value in dts file